src/scripts/main.js

```diff
@@ -355,6 +355,25 @@ const people = [
 ];
 
 // eslint-disable-next-line no-console
-console.log(people); // you can remove it
+// console.log(people);
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const { names, sex, born, died } = person;
```

Comment:
There is a typo in the destructuring assignment. The property 'names' does not exist in the person object. It should be 'name'.